src/praisonai-bot/tests/unit/gateway/test_exec_approval_liveness.py

```
@@ -0,0 +1,109 @@
+    assert res.approved is False
+    assert res.reason == "cancelled"
+    # The request is gone; a later resolve is a no-op.
+    assert mgr.resolve(rid, Resolution(approved=True)) is False
+
+
+def test_stale_resolution_is_dropped(tmp_path):
+    """A resolution arriving after supersede is dropped (fail-closed)."""
+    mgr = ExecApprovalManager(ttl=300, allowlist_path=tmp_path / "allow.sqlite")
+
+    async def go():
+        rid, future = await mgr.register(
```

Ruff (0.16.2)

[warning] Unpacked variable `rid` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)